src/main/java/org/opensearch/knn/index/mapper/KNNVectorFieldMapper.java

```diff
@@ -69,7 +61,7 @@
  */
 public abstract class KNNVectorFieldMapper extends ParametrizedFieldMapper {
 
-    private static Logger logger = LogManager.getLogger(KNNVectorFieldMapper.class);
+    static Logger logger = LogManager.getLogger(KNNVectorFieldMapper.class);
```

Comment:
why not private?

Reply:
I was thinking or re-using same logger in sub-class to keep content of the log file same, as it puts the class name there.
But I on second though it should be a problem as long as we keep the content same. 
Let me update, I also will use lombok for logger initiaslization

Reply:
I would suggest use different loggers for different classes specially for parent and child as sometimes it become difficult to find the right code path.